cxx-squid/src/main/java/org/sonar/cxx/CxxVCppBuildLogParser.java

```diff
@@ -290,24 +319,22 @@ private void ParseCommonCompilerOptions(String line, String fileElement) {
     }
     
     //_DEBUG Defined when you compile with /LDd, /MDd, and /MTd.
-    if (line.contains("/LDd ") ||

```

Comment:
i would rather leave these as they were, since the comments are closer to the code. easier to follow